.circleci/build_image.sh

```diff
@@ -12,13 +12,11 @@ IMAGE_NAME=$2
 
 docker build -t $IMAGE_NAME -f $DOCKERFILE $DIR
 
-if [[ ! -z $DOCKER_USER && ! -z $DOCKER_PASS ]]; then
+if [[ ! -z $DOCKER_USER ]]; then
   # Log in to Docker Hub.
-  docker login -u $DOCKER_USER -p $DOCKER_PASS
-  # Decide whether Docker Hub images should be tagged ":pull-request-X", ":branch-Y" or ":latest".
-  if [[ ! -z $CIRCLE_PULL_REQUEST ]]; then
-    DOCKERHUB_TAG="pull-request-$CIRCLE_PULL_REQUEST"
-  elif [[ $CIRCLE_BRANCH != "master" ]]; then
+  docker login -u $DOCKER_USER --password-stdin <<< $DOCKER_PASS
+  # Decide whether Docker Hub images should be tagged ":branch-Y" or ":latest".
+  if [[ $CIRCLE_BRANCH != "master" ]]; then
```

Comment:
If it's a pull request from a fork's master branch, and credentials are accidentally exposed, won't this override our production images?

Reply:
It doesn't. I mean "exposed" by exposing them in log. This only happens when it's a PR from branch inside this repo.